server/apps/prepopulate/data_initialization/vocabularies.json

```diff
@@ -125,5 +125,22 @@
                 "usageTerms": "The direct recipient must comply with the limitations specified in the AAP Information\r\nSupply Agreement relating to the AAP content including, without limitation, not permitting\r\nredistribution and storage of the content (outside the terms of the Agreement) and not\r\npermitting deep hyperlinking to the content, framing of the content on a web site,\r\nposting the content to usenet newsgroups or facilitating such actions."
             }
         ]
+    },
+    {
+        "_id": "geographical_restrictions",
+        "items": [
```

Comment:
imo, add Western Australia and Northern Territory in vocabularies. That's other half of australia. :)